django_ses/settings.py

```diff
@@ -38,6 +38,7 @@
 AWS_SES_RETURN_PATH_ARN = getattr(settings, 'AWS_SES_RETURN_PATH_ARN', None)
 
 USE_SES_V2 = getattr(settings, 'USE_SES_V2', False)
+AWS_SES_FROM_EMAIL = getattr(settings, 'AWS_SES_FROM_EMAIL', None)
```

Comment:
Yes.

It will cause a breaking change on the following scenery

```python

AWS_SES_RETURN_PATH = "my_default_from_email@email.com"

SESBackend().send_messages(
    [
        {
            "from": "ignored_from_email@email.com",
            "to": "to_email@email.com",
            "subject": "some subject",
            "message": "some message"
        }
    ]
)
```

Before this change, this message would be sent with the `FromEmailAddress=my_default_from_email@email.com`.

After this change, if the attribute `AWS_SES_FROM_EMAIL` is not set with `AWS_SES_FROM_EMAIL=my_default_from_email@email.com`, the email will be sent with the from email of the EmailMessage, which would be `FromEmailAddress=ignored_from_email@email.com`.

The correct usage AFTER THE CHANGE would be

```python

AWS_SES_FROM_EMAIL = "my_default_from_email@email.com"

SESBackend().send_messages(
    [
        {
            "from": "ignored_from_email@email.com",
            "to": "to_email@email.com",
            "subject": "some subject",
            "message": "some message"
        }
    ]
)
```

And this scenery could be a test itself! And it could act as a regression test.

Reply:
If you don't mind, please add this example to the PR description and update the changelog, that will make it easier to merge this PR!